core/src/main/java/org/opensearch/sql/data/model/ExprTimestampValue.java

```diff
@@ -40,7 +36,7 @@ public class ExprTimestampValue extends AbstractExprValue {
   public ExprTimestampValue(String timestamp) {
     try {
       this.timestamp = LocalDateTime.parse(timestamp, DATE_TIME_FORMATTER_VARIABLE_NANOS)
-          .atZone(ZONE)
+          .atZone(UTC_ZONE_ID)
```

Comment:
Is there any interest in including a default time zone in the functionalProperties object?